core/test/integration/synchronization/ethereum_synchronization.cpp

```diff
@@ -268,3 +270,116 @@ TEST_F(EthereumLikeWalletSynchronization, XpubETCSynchronization) {
     }
 }
 
+std::pair<std::shared_ptr<LambdaEventReceiver>, ledger::core::Future<bool>>
+createSyncReceiver() {
```

Comment:
Why not putting that on the same line as above?

Reply:
Done